.env

```diff
@@ -19,15 +16,12 @@ MATVIEW_SQL_DIR=matview_sql
 ES_HOST=http://usaspending-es
 ES_PORT=9200
 
-# ES_CLUSTER_DIR should point to a path outside, or it will be mounted w/Docker
-ES_CLUSTER_DIR=../docker_es
-
 # Configuration values for a connection string to a Broker database
 #    Only necessary for some management commands
 BROKER_USER=root
 BROKER_PASSWORD=password
 BROKER_HOST=broker-db
 BROKER_PORT=5432
 
-# Default to having Django settings.py DEBUG=True for local docker envs
-DJANGO_DEBUG=True
\ No newline at end of file
+# Use env var to toggle Django's DEBUG mode for local docker envs
+DJANGO_DEBUG=False
```

Comment:
I still think the initial default for this ought to be `True`. Pretty much every developer starting out on this project has gotten hung up with local dev envs that "don't start"... and often the reason why is something that Django logs would have brought to light. But the logs settings we have operate on whether DEBUG is true and DEBUG is only true if this env var is true.

Reply:
Good point, but I think the default should be what's most common for all devs. It can be set by new devs by env, then removed when it's working, or updates mess things up again.

Reply:
I don't recall a time debug logs reveal the issue with a local env. Usually the django error output reveals enough info.

Regardless, if DEBUG mode should be enabled as the default I suggest it is set in `settings.py` instead of here so it is consistent across local envs

Reply:
I think that it's not so much an issue of the log _level_ needing to be at debug, but rather redirecting log _output_ to the `console` when developing locally (rather than just to a file). 

I think this part of `settings.py` forces it into the console when otherwise these django error output would otherwise not show up in the console. 

https://github.com/fedspendingtransparency/usaspending-api/blob/0fc869f051ea7c1d56ee58a66d5aef7fdd58c92c/usaspending_api/settings.py#L378-L382

Reply:
i think it's the `server` log that has useful info from Django API which is only sent to `server.log` and not to the console that devs run when invoking `docker-compose`

Reply:
A bit anecdotal, but `DEBUG=False` is a better default after my personal and observed experience of developing locally.